### ISSUE TITLE
BLT 8.9.x + chained command calls do not pass original inputs

### ISSUE THREAD

**Issue by amcgowanca** [acquia/blt#2298]
As part of validation for work I've done for #2297 , I've recently noticed that invoking `blt setup:drupal:install -Dsite=[name]` does not pass the site name to the subsequent commands being invoked as a result of it being defined at the top level. This is a bit problematic for a multisite build. However, running `blt internal:drupal:install -Dsite=[name]` has sufficed for now. Chances are I originally encountered the problem a while ago, but didn't think anything of it until now for validation of the `internal:drupal:install` changes I've done in a replacement command for the purpose of solving #2297 for now.

Specifically...

When running `blt setup:drupal:install -Dsite=[site-name]`, the value from `$this->getConfigValue('site')` in a subsequent command invocation, such as `internal:drupal:install` does not have the `site` value set to which was specified in the option list but instead is set to default.

When running `blt internal:drupal:install -Dsite=[site-name]`, the value from `$this->getConfigValue('site')` is set as expected.

**Comment by danepowell** [comment]
Precisely which version of BLT are you using? It helps if you use the issue template when posting.

I can't reproduce this in 8.9.10. I added this to _DrupalCommand::install()_:

```php
$this->say($this->getConfigValue('site'));
```

I then ran `blt setup:drupal:install -D site=test`

It printed "test", as expected.

By the way, the preferred way to pass parameters now is with a space (i.e. `-D site=test`, not `-Dsite=test`), but I don't think that should affect this issue.
